webapp/channels/src/components/profile_popover/profile_popover.scss

```diff
@@ -1,139 +1,211 @@
-    font-size: 11px;
-    font-weight: 400;
-    letter-spacing: 0.02em;
-    line-height: 16px;
-    text-align: center;
+.user-profile-popover-floating-overlay {
+    // 99 being the z-index of the global header
+    z-index: 100;
 }
 
 .user-profile-popover {
     width: 240px;
     max-height: calc(100vh - 240px);
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+    border-radius: 4px;
+    background: var(--center-channel-bg);
+    box-shadow: var(--elevation-4);
     overflow-y: auto;
 
-    .popover-title {
-        overflow: unset;
-        max-width: 110%;
-        padding: 0;
-        border-bottom: none;
-        margin-top: 8px;
-        margin-right: -8px;
-
-        &.popover-title_height {
-            overflow: visible;
-            height: 22px;
-        }
+    .user-profile-popover-title {
+        display: flex;
+        flex-wrap: nowrap;
+        align-items: center;
+        justify-content: end;
+        padding: 8px 8px 12px 8px;
 
         .user-popover__role {
-            padding: 0 4px;
-            margin-left: 0;
-            background-color: rgba(var(--center-channel-color-rgb), 0.08);
-            color: var(--center-channel-color-rgb);
-            font-family: Open Sans, sans-serif;
-            font-size: 10px;
-            letter-spacing: 0.02em;
-            line-height: 16px;
+            flex-grow: 1;
+            background: unset;
+            padding-inline-start: 8px;
+
+            > span {
+                height: 16px;
+                padding: 0 4px;
+                border-radius: 4px;
+                background-color: rgba(var(--center-channel-color-rgb), 0.08);
+                color: var(--center-channel-color-rgb);
+                font-family: Open Sans, sans-serif;
+                font-size: 10px;
+                letter-spacing: 0.2px;
+                line-height: 16px;
+                text-transform: uppercase;
+            }
         }
     }
 
-    .popover-content {
-        padding: 0 16px;
+    .user-profile-popover-content {
+        .user-popover-image {
+            position: relative;
+            width: 128px;
+            margin: 0 auto 8px auto;
 
-        .overflow--ellipsis {
-            text-align: center;
+            #userAvatar {
+                width: 120px;
+                min-width: 120px;
+                height: 120px;
+            }
+
+            .user-popover-status {
+                position: absolute;
+                top: auto;
+                right: 8px;
+                bottom: 0;
+                display: flex;
+                width: 24px;
+                height: 24px;
+                padding: 2px;
+                border-radius: 50px;
+                background: rgba(var(--center-channel-bg-rgb), 1);
+
+                svg {
+                    width: 100%;
+                    min-height: 100%;
+                }
+            }
         }
 
-        .user-popover__subtitle {
```

Comment:
Could you double check to see if any E2E tests are looking for the old class names for these components?

I also wonder if there's any plugins using these classes. I don't think there's a good way to figure that out without manually testing them all though

Reply:
Yes i will run e2e in QA